pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -718,6 +721,27 @@ func (b *OFBridge) RetryInterval() time.Duration {
 	return b.retryInterval
 }
 
+func (b *OFBridge) PortStatusRcvd(status *openflow15.PortStatus) {
+	if b.portStatusConsumerCh == nil {
```

Comment:
There is a race condition regarding the read and writing of `b.portStatusConsumerCh`.
Note that `OFBridge.Connect` has been called when  `SubscribePortStatusConsumer` is called.

Reply:
Introduced a lock to provide safe guard.